<launch>
   <node name = "line_follower" type="line_follower.py" pkg="lab1" output="screen">
   		<param name="plan_topic" type="string" value="/planner_node/car_plan"/>   		
   		<param name="pose_topic" type="string" value="/sim_car_pose"/>
      <param name="plan_lookahead" type="double" value="1"/>
      <param name="translation_weight" type="double" value="1.0"/>
      <param name="rotation_weight" type="double" value=".5"/>
      <param name="kp" type="double" value="30"/>
      <param name="kd" type="double" value="0.0"/>
      <param name="ki" type="double" value="0.0"/>
      <param name="error_buff_length" type="double" value="10"/>
      <param name="speed" type="double" value="1.0"/>   
   </node>
</launch>